.hound.yml

```diff
@@ -1,4 +1,3 @@
-# Prefer single quotes
-StringLiterals:
-  EnforcedStyle: single_quotes
-  Enabled: true
```

Comment:
Hmm I see. So now it is added into rubocop defaults but not into hound customization. Did you removed lines above manually or it was done when `bundle exec hound-tools` was executed?